render_bundle/tests/unit/src/expected_bundle_sdcore_up.yaml

```
@@ -0,0 +1,17 @@
+bundle: kubernetes
+name: sdcore-up
```

How about writing the full name? `up` might be a bit confusing, it's a word itself and is not an acronym either.
```suggestion
name: sdcore-user-plane
```

---

Agreed, fixed